src/helm/common/request.py

```diff
@@ -59,6 +60,9 @@ class Request:
     if messages is specified for a chat model, the prompt is ignored.
     Otherwise, the client should convert the prompt into a message."""
 
+    multimodal_prompt: Optional[MultimodalContent] = None
```

Comment:
This seems very confusing because I would expect the type to be `multimodal_prompt: Optional[MultimodalPrompt]` but  `MultimodalPrompt` is not `MultimodalContent`...

I feel like `MultimodalPrompt` should be named something else.

Reply:
We discussed this offline. Decided to keep for now.